### ISSUE TITLE
The tab navigation buttons are either hidden or invisible in dark theme

### ISSUE THREAD

**Issue by brazenvoid** [Bionus/imgbrd-grabber#2603]
**Bug description & STRs**

In normal situations these navigation buttons, the left/right tab movers are not there at all in the UI. 

To make them exist, one needs to have tabs enough that they can't be accommodated in the tab bar and then:
- click on the last tab that is at least half visible. 
- any tab that is less visible, you will not be able to get the desired result.
- after clicking and switching to the tab, the two invisible navigation buttons will be inserted in the UI on top of the now last visible tab, towards the end.
- these will have the focus effect but essentially will be transparent. 
- they will be available till you focus something else.

**Expected behaviour**

These buttons must be visible all times or at least when all tabs can't be accommodated/displayed in the tab bar.

**Screenshots**

![1](https://user-images.githubusercontent.com/8722533/154102643-5e50fb6d-06b6-44a8-84b0-e5c94b87fc70.PNG)

![2](https://user-images.githubusercontent.com/8722533/154102687-0cbab9c7-9d20-4357-a925-c07514020459.png)

**System information**

- OS: Windows 10
- Grabber version: 7.7.1

**Comment by brazenvoid** [comment]
The fix doesn't work in the nightly.

**Comment by Bionus** [comment]
> The fix doesn't work in the nightly.

Very likely it's just not getting overwritten. You can try deleting the theme directory and re-installing.
I'll update the installer in the meantime.

**Comment by brazenvoid** [comment]
> > The fix doesn't work in the nightly.
> 
> Very likely it's just not getting overwritten. You can try deleting the theme directory and re-installing. I'll update the installer in the meantime.

Still bad...

**Comment by Bionus** [comment]
I'm not sure what you mean then, now the tab navigation buttons always have their border and solid background?

![image](https://user-images.githubusercontent.com/882719/156247734-83421648-1a06-443d-bf46-ca481a9a5866.png)

**Comment by brazenvoid** [comment]
I meant they should occupy their own space rather than being overlayed on the last tab. 

They can't even be considered proper buttons as they do not elude to their purpose, there are no arrows or indications as to what they do. In your view I see a dot inside but for me they are totally empty black boxes with a space in between for some reason.

If its really CSS then well am really good at it. Gonna take a jab then,

**Comment by Bionus** [comment]
In that case indeed, that's Qt's behavior. Not sure how much can be overridden in CSS for the actual placement of the buttons.
The dots are actually an arrow image ("img/right_arrow.png"), so I guess something could be made to make it clearer though.

**Comment by brazenvoid** [comment]
yeah I tried a lot of things and am still trying but the images aren't getting picked up and it does seem that the CSS being used is limited in its application.
